lib/matplotlib/tests/test_text.py

```diff
@@ -1135,3 +1135,58 @@ def test_font_wrap():
     plt.text(3, 4, t, family='monospace', ha='right', wrap=True)
     plt.text(-1, 0, t, fontsize=14, style='italic', ha='left', rotation=-15,
              wrap=True)
+
+
+def _test_ha_for_angle():
```

Comment:
This function is unused.

Reply:
Hi @QuLogic, it seems like making  `_test_ha_for_angle` and `_test_va_for_angle` private functions causes Codecov to ignore them? Would you like me to remove them? 

I ran these locally, and they pass:
```bash
pytest --pyargs /workspaces/matplotlib/lib/matplotlib/tests/test_axis.py::_test_ha_for_angles
pytest --pyargs /workspaces/matplotlib/lib/matplotlib/tests/test_axis.py::_test_va_for_angles
```